spec/v1.0-draft/digest_set.md

```diff
@@ -53,6 +52,9 @@ Supported algorithms:
 >
 > Two DigestSets SHOULD be considered matching if ANY acceptable field
 > matches.
+>
+> New algorithms must document how the value is encoded. E.g. base64,
```

Comment:
Just an idea (and maybe this has been discussed already): Do we see a downside to adding an optional `encoding` field to the DigestSet type?

Reply:
It's a good thing to discuss.  My first concern is that it would probably be a breaking change?  This would go from map<string, string> to map<string, ValueEntry> where ValueEntry has (encoding, value).  or is there a more clever way to do it?

Reply:
Right, I think people will generally be using the key field as a opaque string, so adding +base64 here just turns it into a new string type (which is still useful!).  Folks will be able to use both goModuleH1 and goModuleH1+base64 if they want.  Maybe in the future we could add fancy support for specifying the encoding in here, but then people would need to handle them as non-opaque strings, which would be a big change.